cfme/configure/tasks.py

```diff
@@ -24,9 +25,12 @@
 
 buttons = Region(
     locators=dict(
-        default="//*[@id='buttons_off']/li[3]/a/img",
-        apply="//*[@id='buttons_on']/li[1]/a/img",
-        reset="//*[@id='buttons_on']/li[2]/a/img"
+        default=pick({LOWEST: "//*[@id='buttons_off']/li[3]/a/img",
+                 '5.4': "//*[@id='buttons_off']/a"}),
+        apply=pick({LOWEST: "//*[@id='buttons_on']/li[1]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[1]"}),
+        reset=pick({LOWEST: "//*[@id='buttons_on']/li[2]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[2]"})
```

Comment:
Are you sure about the indentation? I personally prefer this style in this case:

```python
reset=pick({
    LOWEST: xyz,
    "5.4": xyz,
})
```

Maybe these buttons can be handled with formbutton since the formbuttons were usually placed in the buttons_on/off divs